### PR TITLE
FIX for #185

### PR DESCRIPTION
**Pull request by ClemensSahs**
this will fix the error from the PR #185

### CODE REVIEW

**Comment by ClemensSahs** [comment]
currently only Drupal 8.4 fail

Drupal\taxonomy\Tests\TaxonomyTestTrait is deprecated in Drupal 8.4.0 and will be removed before Drupal 9.0.0. Instead, use \Drupal\Tests\taxonomy\Functional\TaxonomyTestTrait: 1x

File: modules/graphql_views/tests/src/Kernel/ViewsTestBase.php [hunk 1]
```diff
@@ -7,20 +7,18 @@
 use Drupal\simpletest\NodeCreationTrait;
 use Drupal\taxonomy\Entity\Term;
 use Drupal\taxonomy\Entity\Vocabulary;
-use Drupal\taxonomy\Tests\TaxonomyTestTrait;
-use Drupal\Tests\graphql_core\Kernel\GraphQLFileTestBase;
+
```
Comment (fubhy) [inline modules/graphql_views/tests/src/Kernel/ViewsTestBase.php]: Blank line.

File: modules/graphql_views/tests/src/Kernel/ViewsTestBaseDeprecationFix.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Drupal\Tests\graphql_views\Kernel;
+
+use Drupal\Tests\graphql_core\Kernel\GraphQLFileTestBase;
+
+
```
Comment (fubhy) [inline modules/graphql_views/tests/src/Kernel/ViewsTestBaseDeprecationFix.php]: Blank line.

File: modules/graphql_views/tests/src/Kernel/ViewsTestBaseDeprecationFix.php [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+if (version_compare(\Drupal::VERSION, '8.4', '<')) {
+  abstract class ViewsTestBaseDeprecationFix extends GraphQLFileTestBase {
+    use \Drupal\taxonomy\Tests\TaxonomyTestTrait;
+  }
+
```
Comment (fubhy) [inline modules/graphql_views/tests/src/Kernel/ViewsTestBaseDeprecationFix.php]: Blank line.

File: modules/graphql_views/tests/src/Kernel/ViewsTestBaseDeprecationFix.php [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+} else {
+
```
Comment (fubhy) [inline modules/graphql_views/tests/src/Kernel/ViewsTestBaseDeprecationFix.php]: Blank line.

**Review by fubhy — CHANGES_REQUESTED** [review]

**Comment by pmelab** [comment]
Thank you a lot!

**Pull request merged**